arrows/klv/convert_0104_metadata.cxx

```diff
@@ -207,7 +208,7 @@ case klv_0104::KN:                                        \
       CASE( START_DATE_TIME_UTC );
       CASE2( MISSION_START_TIME, MISSION_START_TIME_UTC );
       CASE( SECURITY_CLASSIFICATION );
-      CASE( CLASSIFICATION );
```

Comment:
Lacking context here but what happened to this classification tag?

Reply:
It was a duplicate of `SECURITY_LOCAL_MD_SET`. It appears in both the 0104 (as `CLASSIFICATION`) and 0601 (as `SECURITY_LOCAL_MD_SET`) standards and I guess whoever wrote them didn't realize this at the time. I chose to stick with the `SECURITY_LOCAL_MD_SET` name since it's more descriptive.

Reply:
Thanks, makes sense.